mongo/tests/test_e2e.py

```diff
@@ -97,3 +97,20 @@ def test_e2e_mongo_tls(dd_agent_check):
     for metric in MONGOD_METRICS:
         aggregator.assert_metric(metric)
     aggregator.assert_service_check('mongodb.can_connect', status=MongoDb.OK)
+
+
+@tls
+@pytest.mark.e2e
+def test_e2e_mongo_ssl(dd_agent_check):
```

Comment:
I'd prefer this test to be integration rather than e2e. We don't need an agent image to test this change.

Reply:
Done